gnosis/helper.js

```diff
@@ -0,0 +1,14 @@
+
+async function submitTransactionToGnosisSafe(DEPLOYER_SAFE, artifact, functionName, ...args) {
```

Comment:
We can rename DEPLOYER_SAFE to follow camel case to signify that it's a parameter.

Reply:
done in latest commit